requirements/common.txt

```diff
@@ -512,8 +516,7 @@ zipp==3.1.0 \
     --hash=sha256:aa36550ff0c0b7ef7fa639055d797116ee891440eac1a56f378e2d3179e0320b \
     --hash=sha256:c599e4d75c98f6798c509911d08a22e6c021d074469042177c8c86fb92eefd96 \
     # via importlib-metadata
-json-e==4.0.0 \
-    --hash=sha256:c1add008dbfbb4b7c48d30b42e11e26db4883394989e013b603a63b6c59b8530
```

Comment:
Since this is out of order, I am guessing someone added this manually

Reply:
https://github.com/mozilla/treeherder/commit/3f73a3103ec8fe4b332d388c8db2d562302b6a80#diff-b80ca39e5ba09203c741fcc8e103bd16R515